src/java/org/apache/cassandra/cql3/QueryProcessor.java

```diff
@@ -161,6 +161,12 @@ public static void clearPreparedStatements(boolean memoryOnly)
         if (!memoryOnly)
             SystemKeyspace.resetPreparedStatements();
     }
+    
```

Comment:
fyi: white-space here i had to remove. there was also three lines with trailing whitespace introduced in CompactTableTest, also fixed.